emscripten.py

```diff
@@ -1880,8 +1885,11 @@ def create_runtime_funcs_asmjs(exports, metadata):
   stackMax = stackMax|0;
   STACKTOP = stackBase;
   STACK_MAX = stackMax;
+  tempDoublePtr = STACKTOP;
+  %s
+  STACKTOP = (STACKTOP + 8)|0;
```

Comment:
The stack grows down with the llvm backend.. so should this be -8 in that case?

Reply:
Thanks for reviewing. Agree on the changes, but wanted to just throw this for CI, didn't yet intend for this to need a review.

Actually, is `tempDoublePtr` even used with wasm backend?

Reply:
tempDoublePtr is not needed with the wasm backend (if some code path refers to it there, we should fix that).